cpp/Platform.Random.Tests/RandomHelpersTests.cpp

```diff
@@ -1,10 +1,7 @@
 ﻿namespace Platform::Random::Tests
 {
-    TEST_CLASS(RandomHelpersTests)
+    TEST(RandomHelpersTests, DefaultFieldTest)
     {
-        public: TEST_METHOD(DefaultFieldTest)
-        {
-            Assert.NotNull(RandomHelpers.Default);
-        }
+        ASSERT_NE(RandomHelpers::Default, nullptr);
```

Comment:
**REALLY?**
Okay... We can use... `std::optional<std::mt19937_64>`?
Oh. Non-Kotlin style😡

Reply:
We should compare with default value here, but it will be equal to it. 
Or maybe we can delete this test. 
No need to use `std::optional<std::mt19937_64>` in this case.

Reply:
Default is not default value. 
```cpp
Default { std::random_device{}() };
```

Reply:
Ok, then we can try do like this:
```cpp
ASSERT_NE(RandomHelpers::Default, {});
```

Reply:
Remove this assert